wren-tests/src/test/java/io/wren/testing/TPCH.java

```diff
@@ -0,0 +1,232 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.wren.testing;
+
+import com.google.common.collect.ImmutableMap;
+import com.google.common.io.Resources;
+import io.wren.base.WrenTypes;
+import io.wren.base.dto.Column;
+import io.wren.base.dto.Model;
+
+import java.io.IOException;
+import java.io.UncheckedIOException;
+import java.util.List;
+import java.util.Map;
+
+import static com.google.common.collect.ImmutableList.toImmutableList;
+import static java.lang.String.format;
+import static java.nio.charset.StandardCharsets.UTF_8;
+import static java.util.Objects.requireNonNull;
+import static org.testng.Assert.assertFalse;
+
+public class TPCH
```

Comment:
```suggestion
public class Tpch
```
It would be better.

Reply:
TPCH is an abbreviation. Should keep the upper case is better.